SECP-service/src/main/resources/assets/app/views/login/authenticate.html

```diff
@@ -0,0 +1,17 @@
+<div class="container-fluid">
+    <div class="form-group">
+        <form ng-submit="authenticate()" name="authForm">
+            <h1 id="formHeader">Authenticate</h1>
+            <p>Enter the verification code sent to your email address</p>
```

Comment:
this could be a span to your input. Having it on top your input as a paragraph is not ideal.